src/services/authService/auth.service.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { apiUrl } from '../../config/config';
+
+export const signInApi = (userName, password) => {
+  console.log(`sign in ${userName} ${password}`);
+  return axios.post(`${apiUrl}/auth/login`, { userName, password });
+};
+
+export const signUpApi = (userName, displayName, email, password) => {
+  console.log(`chuj ${displayName} ${userName} ${email} ${password}`);
```

Comment:
🧐

Reply:
byłem w ciemnym miejscu

Reply:
That's quite amusing @mslup 😂🤣 Made my day!!!! 🤣😂😂🤣